sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/MonotonicallyIncreasingID.scala

```diff
@@ -80,7 +80,5 @@ case class MonotonicallyIncreasingID() extends LeafExpression with Stateful {
 
   override def prettyName: String = "monotonically_increasing_id"
 
-  override def sql: String = s"$prettyName()"
```

Comment:
Why this change?

Reply:
It's the default and no need for the override, isn't it?